File: java/Aksesoris.java
public class Aksesoris extends PetShop {
    public Aksesoris(int id, String nama, double harga, int stok,
                   String jenis, String bahan, String warna,
                   String untuk, String size, String merk) {
        super(id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk);
    }
    
    @Override
    public String getJenis() { return super.getJenis(); }
    
    @Override
    public String getBahan() { return super.getBahan(); }
    
    @Override
    public String getWarna() { return super.getWarna(); }
}

File: python/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    def __init__(self, id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk):
        super().__init__(id, nama, harga, stok, jenis, bahan, warna, untuk, size, merk)
    
    def get_jenis(self):
        return super().get_jenis()
    
    def get_bahan(self):
        return super().get_bahan()
    
    def get_warna(self):
        return super().get_warna()